specification/compute/resource-manager/Microsoft.Compute/stable/2020-06-01/examples/GetADedicatedHostGroup.json

```diff
@@ -17,7 +17,60 @@
             {
               "id": "subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/myDedicatedHostGroup/myHostGroup/Hosts/myHost2"
             }
-          ]
+          ],
+          "supportAutomaticPlacement": true,
+          "instanceView": {
+            "hosts": [
+              {
+                "name": "myHost1",
+                "assetId": "eb3f58b8-b4e8-4882-b69f-301a01812407",
```

Comment:
This does not match with your Swagger spec.  Does DedicatedHostInstanceViewWithName contain assetId, availableCapacity, and statuses?  In your PR for Swagger update, DedicatedHostInstanceViewWithName contains name and all of resources

Reply:
The example here is correct. Will update the Swagger spec.